.travis.yml

```diff
@@ -44,6 +44,16 @@ matrix:
       jdk: oraclejdk8
       after_success: bash <(curl -s https://codecov.io/bash) -f target/coverage/codecov.json
 
+  exclude:
+    - env: VERSION=1.9 TARGET=test-cljs
```

Comment:
I think you should also put the explanations here, so people would be able to quickly understand why've setup the build this way.

Reply:
Good point. Added comments there.